compiler-core/src/build.rs

```diff
@@ -301,6 +301,10 @@ pub struct UnqualifiedImport<'a> {
 #[derive(Debug, Clone, PartialEq)]
 pub enum Located<'a> {
     Pattern(&'a TypedPattern),
+    PatternSpread {
+        spread_location: SrcSpan,
+        unused_fields: Vec<(Option<EcoString>, std::sync::Arc<type_::Type>)>,
```

Comment:
Prefer references to taking ownership here, to avoid cloning. All the arguments without modification sounds best to me.

How to use the information can then be up to the engine

Reply:
Ok! I should have fixed it :)